### PR TITLE
Add a status report on CheriBSD.

### PR DESCRIPTION
**Pull request by brooksdavis**

### CODE REVIEW

File: 2020q2/cheribsd.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Contact: Andrew Turner <andrew@FreeBSD.org>
+Contact: Brooks Davis <brooks@FreeBSD.org>
+Contact: Edward Tomasz Napierala <trasz@FreeBSD.org>
+Contact: Jessica Clarke <jrtc27@FreeBSD.org>
+Contact: John Baldwin <jhb@FreeBSD.org>
+Contact: Robert Watson <rwatson@FreeBSD.org>
+Contact: Ruslan Bukin <br@FreeBSD.org>
+
+CheriBSD extends FreeBSD to implement memory protection and software
+compartmentalization features supported by the CHERI ISA.
```
Comment (jrtc27) [inline 2020q2/cheribsd.md:16]: Maybe `CHERI ISAs`/`CHERI instruction set extensions`?

File: 2020q2/cheribsd.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Support for CHERI-RISC-V in CheriBSD has continued to mature this
+quarter in tandem with refinements to the CHERI-RISC-V architecture.
+CheriBSD's "pure capability" (CheriABI) process environment grew support
+for:
```
Comment (jrtc27) [inline 2020q2/cheribsd.md:21]: Do we want to emphasise that this is the native ABI and all of world is built with it, i.e. it's not just an extra thing on the side?

File: 2020q2/cheribsd.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+- dynamically linked binaries (previously only statically-linked binaries were supported)
+- C++ including exceptions
+- sealed return address capabilities ("sentries") which provide additional CFI protection
```
Comment (jrtc27) [inline 2020q2/cheribsd.md:25]: \+ function pointers

File: 2020q2/cheribsd.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+support for CHERI-MIPS.
+
+Much of this effort has been focused on preparing CheriBSD on
+CHERI-RISC-V for inclusion as a demonstrator system in DARPA's Finding
+Exploits to Thwart Tampering ([FETT](https://fett.darpa.mil)) Bug Bounty
+program.
+
+In addition, work has begun this quarter on porting CheriBSD to Arm's
+Morello SoC.  Morello is a prototype demonstrator board which adds CHERI
+extensions to ARMv8-A.
```
Comment (jrtc27) [inline 2020q2/cheribsd.md:38]: Hmm, Armv8-A? The old v8 was before the re-stylising, but newer point revisions use Arm in official marketing, and recent marketing material even retroactively re-stylises older architecture names as Arm (even in Armv7).

Comment (brooksdavis) [reply 2020q2/cheribsd.md:38]: arm.com seems to be mixed up here still referencing a version of the ARM ARM that uses `ARMv8-A`

Comment (jrtc27) [reply 2020q2/cheribsd.md:38]: ITYM the Arm ARM :P

Comment (jrtc27) [reply 2020q2/cheribsd.md:38]: https://developer.arm.com/documentation/ddi0487/latest (if you want to download and open a 50 MiB monster of a PDF) calls it Arm everywhere other than when citing old manuals

**Pull request merged**